### PR TITLE
feat: Add webrtc-w3c protocol for browser to browser

### PR DESCRIPTION
**Pull request by ckousik**
Adds a new protocol called `webrtc-direct` for enabling webrtc browser-to-browser. https://github.com/libp2p/specs/pull/497

### CODE REVIEW

**Comment by mxinden** [comment]
Great. Thanks for providing the patch. I suggest depending on your fork for now, i.e. not release this new multiaddr protocol component yet. That gives us still more time on https://github.com/libp2p/specs/pull/497#discussion_r1083223169.

**Comment by p-shahi** [comment]
Now that `webrtc-w3c` has been adopted per https://github.com/libp2p/specs/pull/497#discussion_r1118904642 are we all clear for merging this PR? cc @mxinden

**Review by mxinden — APPROVED** [review]
All clear to move forward here. I merged https://github.com/multiformats/multiaddr/pull/150 and https://github.com/multiformats/multicodec/pull/316.

**Review by MarcoPolo — APPROVED** [review]

**Comment by p-shahi** [comment]
@achingbrain I think this requires an approval from a maintainer, would you be able to bless it

**Review by achingbrain — APPROVED** [review]

**Pull request merged**
